mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -101,3 +128,141 @@ func ignitionFailure(c cluster.TestCluster) error {
 		return nil
 	}
 }
+
+func ignitionFailure(c cluster.TestCluster) error {
+	// We can't create files in / due to the immutable bit OSTree creates, so
+	// this is a convenient way to test Ignition failure.
+	failConfig, err := conf.EmptyIgnition().Render(conf.FailWarnings)
+	if err != nil {
+		return errors.Wrapf(err, "creating empty config")
+	}
+	failConfig.AddFile("/notwritable.txt", "Hello world", 0644)
+
+	builder := platform.NewQemuBuilder() // platform.Manhole()
+
+	// Create a temporary log file
+	tempLogFile := platform.CreateTempLogFile(builder)
+
+	defer builder.Close()
+	builder.SetConfig(failConfig)
+	err = builder.AddBootDisk(&platform.Disk{
+		BackingFile: kola.QEMUOptions.DiskImage,
+	})
+	if err != nil {
+		return err
+	}
+
+	builder.MemoryMiB = 1024
+	builder.Firmware = kola.QEMUOptions.Firmware
+
+	searchString := "/notwritable.txt"
```

Comment:
I know I suggested this, but one thing with that I just realized is that because Ignition prints the whole config for any error, searching for this doesn't guarantee it failed for the right reason. Maybe let's search for this instead:

> error creating /sysroot/notwritable.txt